docs/_packages/github-repos.md

```diff
@@ -0,0 +1,8 @@
+---
+top:         true
+type:        plugin
+title:       "GitHub repos"
+image:       /images/package-icons/github-repos.png
+githuburl:   gabrielpoca/zazu-github-repos
+description: "Zazu plugin to search and open your GitHub repos"
```

Comment:
Can you remove "zazu" from the sentence? Maybe "Search and open your GitHub repos".